bin/all_sky_search/pycbc_coinc_hdfinjfind

```diff
@@ -170,12 +179,15 @@ for trigger_file, injection_file in zip(args.trigger_files,
     # Put individual detector vetoes into list of all vetoed indices, if an
     # injection is vetoed in N ifos then its index will appear N times
     vetoid = numpy.array([])
+    veto_dict = {}
     for ifo in ifo_list:
         vi, _ = indices_within_segments(inj_time, [args.veto_file], ifo=ifo,
                                         segment_name=args.segment_name)
         vetoid = numpy.append(vetoid, vi)
+        veto_dict[ifo] = vi
         del vi
```

Comment:
If you are now storing the vetodict, the `del vi` should be removed as this would not do anything.

Reply:
Ok, done.